pkg/prometheus/statefulset.go

```diff
@@ -358,6 +353,16 @@ func BuildCommonVolumes(p monitoringv1.PrometheusInterface, tlsAssetSecrets []st
 	return volumes, promVolumeMounts, nil
 }
 
+func ClaimVolumeName(p monitoringv1.PrometheusInterface, cpf monitoringv1.CommonPrometheusFields) string {
```

Comment:
nit: to keep the name consistent with the resource we are referring
```suggestion
func VolumeClaimName(p monitoringv1.PrometheusInterface, cpf monitoringv1.CommonPrometheusFields) string {
```